examples/sidebar-example/content.js

```diff
@@ -1,5 +1,18 @@
 import * as InboxSDK from '@inboxsdk/core';
 
+function addStyle(href) {
```

Comment:
I kind of wonder if this is making the example extension more confusing with this extra stuff. The example extensions don't have to demonstrate every different configuration.

Reply:
We're a long ways from every configuration, but testing that the current change works doesn't have to be something we ultimately commit. I can back this file out if you'd like.

Reply:
yeah, I was just trying to imagine I was someone new to the SDK looking at the examples with the view that they're the concise "right" way to use the basic version of the features. This stuff adds more than a little bit of extra noise that doesn't have to do with the SDK